docs/apiSpecifications/Receipt API.yml

```diff
@@ -106,10 +124,28 @@ paths:
                       example: true
   /reference-data/hazardous-property-codes:
     get:
-      description: Endpoint used to get a list of hazardous property code. It returns a code or code list on <b>200</b> success or a <b>401</b> on failure.
+      description: |
+        Endpoint used to get a list of hazardous property codes. It returns a code or code list on <b>200</b> success or a <b>401</b> on failure. <p></p>

```

Comment:
There is an empty paragraph tag at the end of the line, should this be wrapping the line?

Reply:
Ok, in that case can we use a couple of break tags (`<br><br>`) instead as it's cleaner than using empty paragraph tags?